Description: Fix lintian manpage-has-errors-from-man: .../enchant.1.gz 1: warning: macro `PU' not defined
Author: Agustin Martin Domingo <agmartin@debian.org>
Forwarded: http://bugzilla.abisource.com/show_bug.cgi?id=13163

Index: enchant/doc/enchant.1
===================================================================
--- enchant.orig/doc/enchant.1	2011-10-19 21:58:21.715302260 +0200
+++ enchant/doc/enchant.1	2011-10-19 21:58:58.595483852 +0200
@@ -1,4 +1,3 @@
-.PU
 .TH ENCHANT 1 "July 2006" enchant "Enchant Mini Help File"
 .SH NAME
 Enchant \- a spellchecker
